CryptoAnalysis/src/main/java/crypto/reporting/ErrorMarkerListener.java

```diff
@@ -54,24 +55,44 @@
 	protected final Table<SootClass, SootMethod, Set<AbstractError>> errorMarkers = HashBasedTable.create(); 
 	protected final Map<Class, Integer> errorMarkerCount = new HashMap<Class, Integer>();
 	protected final List<IAnalysisSeed> secureObjects = new ArrayList<IAnalysisSeed>();
+	protected List<String> ignorePackagesList =  Collections.<String>emptyList();
 	
 	private void addMarker(AbstractError error) {
 		SootMethod method = error.getErrorLocation().getMethod();
 		SootClass sootClass = method.getDeclaringClass();
-		
 		Set<AbstractError> set = errorMarkers.get(sootClass, method);
+		Integer integer = errorMarkerCount.get(error.getClass());
 		if(set == null){
 			set = Sets.newHashSet();
 		}
-		if(set.add(error)){
-			Integer integer = errorMarkerCount.get(error.getClass());
-			if(integer == null){
-				integer = 0;
+		if(integer == null){
+			integer = 0;
+		}
+		if(!ignorePackagesList.isEmpty()) {
+			if(!ignorePackagesList.stream().anyMatch((s -> sootClass.getName().contains(s)))) {
```

Comment:
The contains call may result in a result being ignored incorrectly. Imagine "de.darmstadt." being in the list and the name of sootClass is "us.bode.darmstadt.filehandler.". Please fix to startsWith instead of contains.

Reply:
agree. 
also i was expecting some kind of `*` wildcard matching. 
Like if the lists contains: `de.darmstadt.*` this and all sub packages get ignored. Can we do that?

Reply:
Used startsWith instead of contains